monitoring/mongodb/alerts.yaml

```diff
@@ -97,20 +97,65 @@ groups:
   - alert: ReplicationLagWarning
     expr: |
       label_replace(
-        max(
-            max(mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="PRIMARY"})
-                by(pod, rs_nm)
-            - ignoring(member_idx) group_right
-            min(mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="SECONDARY"})
-                by(pod, rs_nm, member_idx)
-        ) by(member_idx, rs_nm) / 1000 > 30
-      , "member_idx", "$1", "member_idx", "(${service}[^.]*)\\.${service}.*")
-    for: 10m
+        max by(rs_nm, member_idx) (
+          (
+            max by(pod, rs_nm) (
+              mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="PRIMARY"}
+            ) - ignoring(member_idx) group_right
+            min by(pod, rs_nm, member_idx) (
+              mongodb_rs_members_optimeDate{namespace="${namespace}",pod=~"${service}.*",member_state="SECONDARY"}
+            )
+          ) / scalar(
+            max by(pod) (
+              (mongodb_mongod_replset_oplog_head_timestamp{namespace="${namespace}",pod=~"${service}.*"}
+              - on(pod)
+              mongodb_mongod_replset_oplog_tail_timestamp{namespace="${namespace}",pod=~"${service}.*"})
+              * on(pod) group_left()
+              (mongodb_mongod_replset_my_state{namespace="${namespace}",pod=~"${service}.*"} == 1)
+            )
+          ) > 0.7
```

Comment:
- should we make thresholds should be configurable, by templatizing them?
- as default values, 70% and 95 seems a little bit weird to me : either we should target something close to limit (95% and 85% maybe), or alert with more margin (70% and 90%) ?